README.md

```diff
@@ -20,6 +20,262 @@ Or install it yourself as:
 
 ## Usage
 
+
+### A simple CREATE, INSERT and SELECT, with raw SQL
+
+```ruby
+require 'async'
+require 'db/client'
+require 'db/postgres'
+
+Console.logger.level = :info
+
+client = DB::Client.new(DB::Postgres::Adapter.new(
+    database: 'test',
+    host:     '172.17.0.3',
+    password: 'test',
+    username: 'postgres',
+))
+
+Async do
+    session = client.session
+
+    create = "CREATE TABLE IF NOT EXISTS my_table (a_timestamp TIMESTAMP NOT NULL)"
+    session.query(create).call
+
+    insert = "INSERT INTO my_table VALUES (NOW()), ('2022-12-12 12:13:14')"
+    session.query(insert).call
+
+    result = session.query("SELECT * FROM my_table").call
+
+    Console.logger.info result.field_types.to_s
+    Console.logger.info result.field_names.to_s
+    Console.logger.info result.to_a.to_s
+
+ensure
+    session&.close
+end
+
+=begin
+ 0.01s     info: [#<DB::Postgres::Native::Types::DateTime:0x00007eff3b13e688 @name="TIMESTAMP">]
+ 0.01s     info: ["a_timestamp"]
+ 0.01s     info: [[2022-11-18 00:54:56.814753 UTC], [2022-12-12 12:13:14 UTC]]
+=end
+```
+
+### Parameterized CREATE, INSERT and SELECT
+The same process as before, but parameterized.
+Always used the parameterized form when dealing with untrusted data.
+
+```ruby
+# ...
+
+Async do
+    session = client.session
+
+    session.clause("CREATE TABLE IF NOT EXISTS")
+        .identifier(:my_table)
+        .clause("(")
+            .identifier(:a_timestamp).clause("TIMESTAMP NOT NULL")
+        .clause(")")
+        .call
+
+    session.clause("INSERT INTO")
+        .identifier(:my_table)
+        .clause("VALUES (")
+            .literal("NOW()")
+        .clause("), (")
+            .literal("2022-12-12 12:13:14")
+        .clause(")")
+        .call
+
+    result = session.query("SELECT * FROM my_table").call
```

Comment:
You should use `identifier` here for the table name.

Reply:
Good catch. Fixed.